public/scripts/guardian_display_handler.js

```diff
@@ -0,0 +1,41 @@
+var guardian_display_handler = (function(response, target_node){
+
+    const base = base_display_handler;
+
+  
+    /* make elements creates the paragraph elements */
+    function make_elements(obj){
```

Comment:
This method is only called when you are calling create_thumbnail. Basically, you need that method to create arguments in the right format for create_thumbnail.
A better option would be to pass `obj` to `create_thumbnail` directly, and then call `make_elements` on `obj` inside `create_thumbnail`